shared/common-adapters/confirm-modal/index.js

```diff
@@ -0,0 +1,126 @@
+// @flow
+import * as React from 'react'
+import {Box, Box2, ButtonBar, HeaderOrPopup, Icon, Text, WaitingButton} from '..'
+import * as Styles from '../../styles'
+import type {IconType} from '../icon.constants'
+
+// generally one of icon or header will be given
+export type Props = {|
+  confirmText?: string,
+  content?: React.Node,
+  description: string,
+  header?: React.Node,
+  icon?: IconType,
+  onCancel: () => void,
+  onConfirm: () => void,
+  prompt: string,
+  waitingKey?: string,
+|}
+
+class _ConfirmModal extends React.Component<Props> {
```

Comment:
Not a big deal, but is there a reason to use `React.Component` over `React.PureComponent`? This seems to be pure. Could also be a function component I suppose.

Reply:
not at all, just copy and paste, will try changing it

Reply:
no